mcda/utils.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import argparse
+import json
+
+
+
+def read_input_matrix(input_matrix_path: str) -> pd.DataFrame():
+    try:
+        with open(input_matrix_path, 'r') as fp:
+            return pd.read_csv(fp, sep=';')
+    except Exception as e:
+        print(e)
```

Comment:
Should logger be used here as well similar to what you used in `mcda_sensitivity_analysis.py` instead of `print` statements?

Reply:
Here I am raising an exception and not just giving the user an info message. The block `try - except - print` is commonly used in Python.

However, it is true that one can also use something like:
```
logger.error('Error Message', stack_info=True)
raise ValueError('custom error message')
```
but not with the `try` logic